spec/spec_helper.rb

```diff
@@ -20,6 +20,7 @@
   config.include Rack::Test::Methods
   config.include Spec::Support::Helpers
   config.raise_errors_for_deprecations!
+  config.filter_run_when_matching :focus
```

Comment:
This line is out of the scope from this PR,  but I find it to be very useful. Can we keep?